sync_packages.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+robot_name="${HOSTNAME//-b1}"
+
+if [ "$HOSTNAME" != "$robot_name-b1" ]; then 
+	echo "FATAL: CAN ONLY BE EXECUTED ON BASE PC"
+	exit
+fi
+
+sudo apt list --installed > /tmp/package_list
+perl -pi -e 's{ / .*? ]}{}x' /tmp/package_list
+sed -i 's/Listing...//' /tmp/package_list 
+sed -i ':a;N;$!ba;s/\n/ /g' /tmp/package_list
+tr '\n' ' ' < /tmp/package_list | sed -i '$s/ $/\n/' /tmp/package_list
+packages=$(cat /tmp/package_list)
+
+pcs="
+$robot_name-b1
+$robot_name-t1
+$robot_name-t2
+$robot_name-t3
+$robot_name-s1
+$robot_name-h1"
```

Comment:
well the name of the repository is setup_cob4... I usually use this repository only for cob4 specific setup. I can check if it can be easily parametrizable, but then I have to modify all the scripts of this repo...